optuna_dashboard/ts/components/StudyHistory.tsx

```diff
@@ -101,6 +102,15 @@ export const StudyHistory: FC<{ studyId: number }> = ({ studyId }) => {
           />
         </CardContent>
       </Card>
+      <Card
+        sx={{
+          margin: theme.spacing(2),
+        }}
+      >
+        <CardContent>
+          <GraphTimeline study={studyDetail} />
+        </CardContent>
+      </Card>
```

Comment:
How about moving this into `Grid2` container with `xs={6}` since this graph is still new and the `GraphHyperparameterImportance` plot is more important for most of users?

Reply:
Fixed.